packages/kit/src/runtime/client/router.js

```diff
@@ -88,7 +88,8 @@ export class Router {
 					'sveltekit:scroll': scroll_state()
 				};
 				history.replaceState(new_state, document.title, window.location.href);
-			}, 50);
+			// iOS scroll event intervals happens between 30-150ms, sometimes around 200ms
```

Comment:
I just noticed an extra s in the text I copied from the PR description :smile: 
```suggestion
			// iOS scroll event intervals happen between 30-150ms, sometimes around 200ms
```

Reply:
My grammar :see_no_evil: Fixed in the latest commit